src/Robo/Plugin/Commands/DevelopmentModeBaseCommands.php

```diff
@@ -124,10 +127,17 @@ public function databaseRefreshTugboat(): Result
                 ->run();
             $result = $this->taskExec('rm')->args($dbPath)->run();
 
-            $result = $this->taskExec("$this->vendorDirectory/bin/drush")
-                ->arg('cache:rebuild')
-                ->dir("$this->drupalRoot/sites/$siteName")
-                ->run();
+            if ($drupal7) {
```

Comment:
@markdorison Should the if/else logic be reversed. Here it says use `drush cr` if we're on D7, which should be `drush cc all`.

Reply:
Another issue is `$siteName` because that depends on the key in `.sites.config.yml` and that doesn't necessarily match the site folder names. (E.g. `intuit` vs `intuitbenefits.com`)

Reply:
@mmatsoo Let's hop on a quick zoom. It might be better to just create a custom command for this scenario.